spec/devise_saml_authenticatable/saml_mapped_attributes_spec.rb

```diff
@@ -8,8 +8,10 @@
   let(:saml_attributes) do
     {
```

Comment:
It looks like this should be `OneLogin::RubySaml::Attributes.new(` instead of just a hash. Not sure exactly how to construct one off the top of my head, so definitely double check the syntax 🙂

Reply:
Fixed, and also did the change in isolation - https://github.com/apokalipto/devise_saml_authenticatable/pull/248 - probably worth landing that first if the build is green.